samples/src/main/java/org/ishugaliy/allgood/consistent/hash/samples/WorkloadBenchmarkSandbox.java

```diff
@@ -1,16 +1,61 @@
-package org.ishugaliy.consistent.hash.samples;
-
-import org.ishugaliy.consistent.hash.ConsistentHash;
-import org.ishugaliy.consistent.hash.HashRing;
-import org.ishugaliy.consistent.hash.hasher.DefaultHasher;
-import org.ishugaliy.consistent.hash.hasher.Hasher;
-import org.ishugaliy.consistent.hash.node.SimpleNode;
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2020 Iurii Shugalii
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.ishugaliy.allgood.consistent.hash.samples;
+
+import org.ishugaliy.allgood.consistent.hash.ConsistentHash;
+import org.ishugaliy.allgood.consistent.hash.HashRing;
+import org.ishugaliy.allgood.consistent.hash.hasher.DefaultHasher;
+import org.ishugaliy.allgood.consistent.hash.hasher.Hasher;
+import org.ishugaliy.allgood.consistent.hash.node.Node;
+import org.ishugaliy.allgood.consistent.hash.node.SimpleNode;
 
 import java.util.List;
 import java.util.UUID;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+/**
+ * Sandbox allows to check consistent hash performance.
+ * Showing how execution time depends on on the ring size and partition rate.
```

Comment:
* Sandbox allows checking consistent hash performance.
 * Showing how execution time depends on the ring size and partition rate.